client/src/pages/BuyerDashboard.js

```diff
@@ -9,6 +10,7 @@ export function BuyerDashboard() {
 				<Link to="/publish-tender">
 					<button>Publish Tender</button>
 				</Link>
```

Comment:
Remove all buttons and leave only `tenders` as title

```suggestion
				<h1>Tenders</h1>
```

Reply:
what about the publish button? do i remove it from the dashboard?

Reply:
Remove the button and only leave the tenders title